systems/robotInterfaces/QPControllerPlan.m

```diff
@@ -1,4 +1,4 @@
-classdef QPControllerPlan
+classdef QPControllerPlan < handle
```

Comment:
It's big, but it's necessary. For toe-off behavior, plans need the ability to modify themselves. We could do that by introducing a SharedDataHandle, but that would create a weird separation between properties which pass by reference and those which pass by value. I think this is a clearer indication of intent.